PCAxis.Sql/PCAxis.Sql.csproj

```diff
@@ -46,8 +46,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="2.19.110" />
-    <PackageReference Include="PCAxis.Core" Version="1.2.2" />
-    <PackageReference Include="PCAxis.Encryption" Version="1.0.2" />
+    <PackageReference Include="PCAxis.Core" Version="1.1.0" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="7.0.1" />

```

Comment:
PCAxis.Core downgrade on purpose?

Reply:
No looks like a misstake. I corrected it https://github.com/statisticssweden/PCAxis.Sql/blob/dfe28e0000a8dc368d2fcdadd7b1b25ea97ef753/PCAxis.Sql/PCAxis.Sql.csproj#L49 the PR still looks like it is still 1.1.0.